### PR TITLE
Sol 294 dashboard cases vs deaths

### PR DESCRIPTION
**Pull request by Alastair-L**
### Issue #: SOL-294

Also includes a small rename.

(test data)
image

### CODE REVIEW

File: packages/database/src/migrations/20210810063133-AddFijiCOVIDDashboardCasesVsDeaths-modifies-data.js [hunk 1]
```diff
@@ -0,0 +1,198 @@
...
+  code: `FJ_Covid_Cases_Vs_Deaths`,
+  reportConfig: {
+    fetch: {
+      aggregations: [
+        {
+          type: 'RAW',
+          config: {
+            dataSourceEntityType: 'sub_district',
+          },
+        },
```
Comment (billli0) [inline packages/database/src/migrations/20210810063133-AddFijiCOVIDDashboardCasesVsDeaths-modifies-data.js:52]: Same as Rohan's feedback, it is optional👍 https://github.com/beyondessential/tupaia/pull/2984#discussion_r688164387

File: packages/indicators/src/Builder/AnalyticArithmeticBuilder/AnalyticArithmeticBuilder.ts [hunk 2]
```diff
@@ -93,11 +93,11 @@ export class AnalyticArithmeticBuilder extends Builder {
   };
 
   private fetchFormulaAnalytics = async (fetchOptions: FetchOptions) => {
-    const aggregationLisByElement = stripFields(
+    const aggregationListByElement = stripFields(
```
Comment (billli0) [inline packages/indicators/src/Builder/AnalyticArithmeticBuilder/AnalyticArithmeticBuilder.ts:96]: Nice catch!!🙌

**Review by billli0 — APPROVED** [review]
Looks good to me! Welcome back to Tupaia @Alastair-L 🎉

File: packages/database/src/migrations/20210810063133-AddFijiCOVIDDashboardCasesVsDeaths-modifies-data.js [hunk 3]
```diff
@@ -0,0 +1,198 @@
...
+        COVIDDeath_FJNumDeaths: 'sum',
+        COVID_FJ_Positive_Tests_By_Sub_District: 'sum',
+      },
+      {
+        transform: 'select',
+        "'Deaths'": '$row.COVIDDeath_FJNumDeaths',
+        "'Cases'": '$row.COVID_FJ_Positive_Tests_By_Sub_District',
+        "'name'": "periodToDisplayString($row.period, 'DAY')",
+        "'timestamp'": 'periodToTimestamp($row.period)',
+        '...': [],
```
[the remark this answers is not in the document]
Comment (Alastair-L) [reply packages/database/src/migrations/20210810063133-AddFijiCOVIDDashboardCasesVsDeaths-modifies-data.js:71]: So clean

File: packages/database/src/migrations/20210810063133-AddFijiCOVIDDashboardCasesVsDeaths-modifies-data.js (hunk 1 above)
Comment (Alastair-L) [reply packages/database/src/migrations/20210810063133-AddFijiCOVIDDashboardCasesVsDeaths-modifies-data.js:52]: Thanks... I think you are right but after spending 10 minutes on it I don't think it's worth investing any more time, I'll leave it this time but keep it in mind 🙏.

**Pull request merged**